docs/apm/service-overview.asciidoc

```diff
@@ -119,14 +124,20 @@ requires an agent version ≥ v5.6.3.
 
 [role="screenshot"]
 image::apm/images/spans-dependencies.png[Span type duration and dependencies]
+// end::dependencies[]
 
 [discrete]
 [[service-cold-start]]
 === Cold start rate
 
-The cold start rate chart is specific to serverless services.
-It displays the percentage of requests that trigger a cold start of a serverless function.
-See <<apm-lambda-cold-start-info>> for more information.
+The cold start rate chart is specific to serverless services, and displays the
+percentage of requests that trigger a cold start of a serverless function.
+A cold start occurs when a serverless function has not been used for a certain period of time.
+Analyzing the cold start rate can be useful for deciding how much memory to allocate to a function,
+or when to remove a large dependency.
+
+The cold start rate chart is currently supported for <<apm-lambda-cold-start-info,AWS Lambda>>
+functions and Azure functions.
```

Comment:
These changes address https://github.com/elastic/observability-docs/issues/2630. I'm curious to know if we want a dedicated page for Azure functions like we already have for [Lambda functions](https://kibana_153140.docs-preview.app.elstc.co/guide/en/kibana/master/apm-lambda.html). There would be a lot of overlap, which is why I didn't create a new page. Does it instead make sense to turn the lambda page into a generic serverless page?